js_modules/dagit/packages/core/src/assets/PartitionHealthSummary.tsx

```diff
@@ -34,7 +34,7 @@ export const PartitionHealthSummary: React.FC<{
     return <div />;
   }
 
-  const keys = timeDimension.partitionKeys;
+  const keys = assetData.dimensions[0].partitionKeys;
```

Comment:
Hmm do you remember if this was broken before?

I think there are a couple other places in Dagit where we identify the "isTimeseries" partition and then use that for a timeline (it's kind of annoying but right now PartitionStatus.tsx is a separate version of a component very similar to this one, but it's interactive). If this works when it uses `timeDimension.partitionKeys`, it's probably preferable to continue doing that for now for consistency with those other places.

Josh and I were talking about revisiting this and making a variant of the timeline for static partition sets, possibly a list of boxes where there's a small whitespace between them marking that it's not really a "range".

Reply:
Yep, this was broken before. So this PR changes the default partition bar view and folder view to choose whichever dimension was ordered first. If the first dimension is not the time dimension, the partition view would break and the whole display would not show up before I made this change.

Reply:
Ok that's good to know! I'll do a bit more testing of this case today just to be safe